com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -256,8 +256,8 @@ private void DrawFullscreenMesh(CommandBuffer cmd, Material material, int passIn
         {
             if (m_UseDrawProcedural)
             {
-                Vector4 scaleBias = new Vector4(1, 1, 0, 0);
-                cmd.SetGlobalVector(ShaderPropertyId.scaleBias, scaleBias);
+                Vector4 scaleBiasSrcUV = new Vector4(1, 1, 0, 0);
```

Comment:
for src uv can't we use UV_STARTS_AT_TOP and/or projectionParams? 

https://docs.unity3d.com/Manual/SL-PlatformDifferences.html

Reply:
I will investigate this tomorrow.